test/vstest.console.UnitTests/Processors/EnableDiagArgumentProcessorTests.cs

```diff
@@ -24,10 +24,26 @@ public class EnableDiagArgumentProcessorTests
 
         private readonly Mock<IFileHelper> mockFileHelper;
 
+        private TraceLevel traceLevel;
+        private string traceFileName;
+
         public EnableDiagArgumentProcessorTests()
         {
             this.mockFileHelper = new Mock<IFileHelper>();
             this.diagProcessor = new TestableEnableDiagArgumentProcessor(this.mockFileHelper.Object);
+
+            // Saving the EqtTrace state
+            traceLevel = EqtTrace.TraceLevel;
+            EqtTrace.TraceLevel = TraceLevel.Off;
+            traceFileName = EqtTrace.LogFile;
+        }
+
+        [TestCleanup]
+        public void Cleanup()
+        {
+            // Restoring to initial state for EqtTrace
+            EqtTrace.TraceLevel = traceLevel;
```

Comment:
Change the sequence of these statements. TraceLevel is set inside InitializeVerboseTrace.